definitions/contested/json/CaseEventToComplexTypes/DFR-3332-1-nonprod.json

```diff
@@ -64,16 +84,27 @@
     "CaseEventID": "FR_approveOrders",
     "CaseFieldID": "judgeApproval1",
     "ListElementCode": "judgeDecision",
+    "EventElementLabel": "Is this ${judgeApproval1.inlineDocType} ready to be sealed and issued?",
     "DisplayContext": "MANDATORY",
     "FieldShowCondition": "judgeApproval1.title=\"*\" AND judgeApproval1.hearingInfo=\"*\""
   },
+  {
+    "LiveFrom": "29/10/2024",
+    "ID": "FR_approveOrdersToJudgeApproval1",
+    "CaseEventID": "FR_approveOrders",
+    "CaseFieldID": "judgeApproval1",
+    "EventElementLabel": " ",
+    "ListElementCode": "amendedDocument",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "judgeApproval1.title=\"*\" AND judgeApproval1.judgeDecision=\"JUDGE_NEEDS_TO_MAKE_CHANGES\""
+  },
   {
     "LiveFrom": "29/10/2024",
     "ID": "FR_approveOrdersToJudgeApproval1",
     "CaseEventID": "FR_approveOrders",
     "CaseFieldID": "judgeApproval1",
     "ListElementCode": "isFinalOrder",
     "DisplayContext": "OPTIONAL",
-    "FieldShowCondition": "judgeApproval1.title=\"Draft Order\" AND judgeApproval1.judgeDecision=\"READY_TO_BE_SEALED\""
+    "FieldShowCondition": "judgeApproval1.docType=\"DRAFT_ORDER\" AND (judgeApproval1.judgeDecision=\"READY_TO_BE_SEALED\" OR judgeApproval1.judgeDecision=\"JUDGE_NEEDS_TO_MAKE_CHANGES\")"
   }
 ]
```

Comment:
add a new line, repeat for the other files.

Reply:
![Screenshot 2024-12-11 at 12 28 43](https://github.com/user-attachments/assets/861457de-d00c-4bd3-a587-ba5ddba437bd)

new line is already here